webui/react/src/components/kit/LineChart.tsx

```diff
@@ -124,6 +125,18 @@ export const LineChart: React.FC<Props> = ({
     return [xValues, ...yValuesArray];
   }, [series, xAxis]);
 
+  const xTicksAdjusted: uPlot.Axis.Values | undefined = useMemo(
+    () =>
+      xTickValues ??
```

Comment:
once we remove the feature flag, can we remove `xTickValues` from the props and just do this internally?

in the meantime `xTicksAdjusted` seems a little misleading because if the user passes `xTickValues` we are just leaving them.

Reply:
TrialDetails/Profiles is the one place using `xTickValues` and `yTickValues` props, so I probably could just move it here, I didn't know if there was future interest in keeping it

Reply:
ok the last commit  removes the `xTickValues` prop, this only affects the Profiler, example is  http://localhost:3000/det/experiments/1788/profiler?f_chart=on

then I renamed `xTicksAdjusted` as the new `xTickValues`

we're still accepting `yTickValues` as a prop from Profiler; I can make that into a separate ticket